tests/benchmarks/README.md

```diff
@@ -0,0 +1,13 @@
+# Automated Benchmarking
+[Benchmark repo](https://github.com/rlworkgroup/benchmarks)
+
+We have an automated system that will run any benchmark file named "test\_benchmark_*.py" in this folder /tests/benchmarks/.
+
+The results are displayed [here](https://rlworkgroup.github.io/benchmarks/)
+
+# Directions
+    Follow these directions to output your benchmark results as a json compatible with the display.
+    1. import tests.helpers as Rh
```

Comment:
i think this would be more clear in a code block https://help.github.com/en/articles/creating-and-highlighting-code-blocks